app.go

```diff
@@ -55,6 +56,9 @@ func createServer() (s *http.Server) {
 	// append env variables to config.Env
 	config.AppendEnvironment(config.Env)
 
+	//connect to monogoDB and select database
+	database.NewMongoDBClient(config.Env.MongoDBUri, config.Env.MongoDBName)
+
 	port := fmt.Sprintf(":%s", config.Env.Port)
```

Comment:
Can you move this Sprintf to line 65 so that we save memory and we get 
```go
	s = &http.Server{
		Addr:           fmt.Sprintf(":%s", config.Env.Port),
		ReadTimeout:    10 * time.Second,
		WriteTimeout:   10 * time.Second,
		MaxHeaderBytes: 1 << 20,
		Handler:        logger,
	}
```